### DISCUSSION TITLE
Minimum requirements for the most advanced bot in ParlAI without GPU

### DISCUSSION THREAD

**Discussion by jacek-michalak** [facebookresearch/ParlAI#4803] [General]
parlai i -mf zoo:seeker/r2c2_blenderbot_3B/model

Ubuntu 20.04.4
16 GB RAM
16 GB swap file

Loading time: 12 min.
The bot response time to the question is about a minute.

**Comment by mojtaba-komeili** [comment]
Yeah, I tried this on my MacBook with 32 GB of RAM. It loaded faster, but the response time was definite around 1-2 minutes.
Thanks for adding to the comments here.

**Reply by jacek-michalak** [reply]
What is exactly the loading time at 32 GB RAM? I'm thinking about expanding the RAM on my laptop

**Reply by mojtaba-komeili** [reply]
I didn't really major it an I believe it depends a lot on What else is going on on your computer. For example, I loaded mine using Docker.
